src/main/php/PDepend/Metrics/Analyzer/ClassLevelAnalyzer.php

```diff
@@ -401,6 +399,9 @@ private function calculateWmci(AbstractASTType $type)
         return $ccn;
     }
 
+    /**
+     * @param ASTClass|ASTEnum $class
```

Comment:
I guess it could be anything here no? Why do we narrow the type to those 2?
```suggestion
     * @param ASTClass|ASTEnum|ASTTrait|\PDepend\Source\AST\ASTAnonymousClass $class
```

Reply:
Silly men they extend ASTClass.